File: 151004/Bordilovsky/DC_Kafka&Redis/Discussion/Services/Mappers/NoteMapper.cs
using AutoMapper;
using Discussion.DTOs.Request;
using Discussion.DTOs.Response;
using Discussion.Entities;

namespace Discussion.Services.Mappers
{
	public class NoteMapper: Profile
	{
		public NoteMapper() 
		{
			CreateMap<Note, NoteResponseTo>();
			CreateMap<NoteRequestTo, Note>();

			CreateMap<Note, NoteRequestTo>();
		}
	}
}

File: 151001/Manchik/discussion/src/main/java/discussion/mappers/NoteMapper.java
package discussion.mappers;

import discussion.dto.NoteRequestTo;
import discussion.dto.NoteResponseTo;
import discussion.entities.Note;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface NoteMapper {
    Note toNote(NoteRequestTo note);
    NoteResponseTo toNoteResponse(Note note);
}
